Task1/Task1/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение x");
+            var x = double.Parse(Console.ReadLine());
+
+            Console.WriteLine($"f(x) = {Function(x):F3}");
+
+            Console.ReadKey();
+        }
+
+        static double Function(double x)
+        {
+            return Math.Sqrt(1 + Math.Cos(x));
+        }
```

Comment:
Лучше использовать содержательные имена параметров
```suggestion
        static double Function(double angle)
        {
            return Math.Sqrt(1 + Math.Cos(angle));
        }
```